freestyle/shared/src/main/scala/freestyle/module.scala

```diff
@@ -98,6 +102,10 @@ object openUnion {
       q"""new {
         ..$parsed
       }"""
+
+    if (showTrees)
+      c.echo(c.enclosingPosition, s"generated tree:\n${showCode(expr)}")
+
```

Comment:
As far as I know this `openUnion` macro is only called by the `@module` macro, so that means that we will show this tree twice (which might be confusing).